access/src/main/java/edu/unc/lib/dl/ui/controller/FullRecordController.java

```diff
@@ -162,8 +163,13 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
 
         AccessGroupSet principals = getAgentPrincipals().getPrincipals();
 
-        aclService.assertHasAccess("Insufficient permissions to access full record",
-                pid, principals, Permission.viewMetadata);
+        String errorMsg = "Insufficient permissions to access full record";
+        try {
+            aclService.assertHasAccess(errorMsg,
+                    pid, principals, Permission.viewMetadata);
+        } catch (AccessRestrictionException e) {
+            LOG.info("{}. {}", errorMsg, e.getMessage(), e);
+        }
```

Comment:
This is eating the assertHasAccess call, so not a whole lot of purpose to it now. Also, this is still logging a full stacktrace for me on permission failures. I'd suggest a few changes, including throwing an exception, not passing the exception into the log statement
```suggestion
        try {
            aclService.assertHasAccess("Insufficient permissions to access full record",
                    pid, principals, Permission.viewMetadata);
        } catch (AccessRestrictionException e) {
            LOG.info("{}", e.getMessage());
            throw new InvalidRecordRequestException();
        }
```

Reply:
Also, can you make these same changes up in the `/{pid}/metadataView` end point in this same class? It has the same problem if you access it directly